cran-comments.md

```diff
@@ -0,0 +1,11 @@
+## Test environments
+* local R installation, macOS, R 4.1.0
+* rhub Ubuntu Linux, R 4.1.0
+* rhub Fedora Linux, R 4.1.0
+* win-builder (devel)
+
+## R CMD check results
+
+0 errors | 0 warnings | 1 note
+
+* This is a new release.
```

Comment:
Need to update this with notes about what was fixed and how for CRAN. One bullet point per item raised in the email.

Reply:
I have added notes about fixes for CRAN in the file. Please let me know if you would like me to change the notes.